### PR TITLE
Math-Magicians Full-website

### PR DESCRIPTION
**Pull request by 1GORDON**
# In this project, I achieved the following

- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Used React basic styling.
- Used React-Router to make the routes work.

### CODE REVIEW

File: src/pages/Mathematics.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import Calculator from '../components/Calculator.js';
```
Comment (Ghiftee) [inline src/pages/Mathematics.js]: - [x] The reason for the error above is you are importing components/Calculator.js but you should be importing components/calculator.js.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽  @1GORDON 

Good job working on the project so far! 
However, there are some issues that you need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [x] When I run your project, I get this error:
Screenshot 2021-12-01 at 19 05 37
You can see the reason for this error in the comment below 👇🏽 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/pages/Mathematics.js (hunk 1 above)
Comment (1GORDON) [reply src/pages/Mathematics.js]: But I don't have calculator.js.
I instead have Calculator.js. And it runs fine on my side

Comment (1GORDON) [reply src/pages/Mathematics.js]: Fixed

Comment (Emmyn5600) [reply src/pages/Mathematics.js]: Good Job

fullsite

**Review by Emmyn5600 — APPROVED** [review]
STATUS: APPROVED :+1: 

Hello, the team @1GORDON   👋🏾 👋🏾 congratulations on making it so far on this milestone.

## Optional suggestions

- Kindly, work on the readme file, add screenshot images, live link demo, and your social accounts.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

**Pull request merged**